library/oneview_appliance_device_snmp_v3_trap_destinations.py

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+###
+# Copyright (2016-2017) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+ANSIBLE_METADATA = {'status': ['stableinterface'],
+                    'supported_by': 'community',
+                    'metadata_version': '1.1'}
+
+DOCUMENTATION = '''
+---
+module: oneview_appliance_device_snmp_v3_trap_destinations
+short_description: Manage the Appliance Device SNMPv3 Trap Destinations.
+description:
+    - Provides an interface to manage the Appliance Device SNMPv3 Trap Destinations.
+version_added: "2.7"
+requirements:
+    - "python >= 2.7.9"
+    - "hpOneView >= 4.8.0"
+author:
+    "Gianluca Zecchi (@gzecchi)"
+options:
+    state:
+        description:
+          - Indicates the desired state for the Appliance Device SNMPv3 Trap Destinations.
+            C(present) ensures data properties are compliant with OneView.
+            C(absent) removes the resource from OneView, if it exists.
+        choices: ['present', 'absent']
+    data:
+        description:
+            - List with the SNMPv3 Trap Destinations properties
+        required: true
+
+extends_documentation_fragment:
+    - oneview
+'''
+
+EXAMPLES = '''
+- name: Ensure that the SNMPv3 Trap Destination is present
+  oneview_appliance_device_snmp_v3_trap_destinations:
+    config: "{{ config }}"
```

Comment:
its better to replace the config with the actual parameters like below so that we can sync with Ansible core
hostname: 172.16.101.48
username: administrator
password: my_password
api_version: 800

Reply:
Fixed in commit 03c17476c328225d849c5a015658949e0f501a5e